blues/application/tasks.py

```diff
@@ -224,24 +224,54 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository()
+    if 'url' not in url or not url.startswith('git@github.com:'):
```

Comment:
What is? A `git_repository` might not be defined for something that the deploy hook, or that function, is used for in the future, and we only support GitHub URLs.